testing/test_junitxml.py

```diff
@@ -944,12 +955,12 @@ class FakeConfig:
         if TYPE_CHECKING:
             workerinput = None
 
-        def __init__(self):
+        def __init__(self) -> None:
             self.pluginmanager = self
             self.option = self
             self.stash = Stash()
 
-        def getini(self, name):
+        def getini(self, name: object) -> str:
```

Comment:
```suggestion
        def getini(self, name: str) -> str:
```

maybe? even if `object` is technically correct because we don't need it, `str` seems less confusing.